docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -130,14 +130,14 @@ The connector supports pushdown for a number of operations:
 * :func:`min`
 * :func:`sum`
 
-Limitations
+.. _clickhouse-sql-support:
+
+SQL support
 -----------
 
-The following SQL statements aren't  supported:
+The connector provides read access to data and metadata in
+a ClickHouse catalog. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
 
-* :doc:`/sql/delete`
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
-* :doc:`/sql/show-roles`
-* :doc:`/sql/show-role-grants`
+* :ref:`sql-schema-table-management`
```

Comment:
I thinking linking instead of inline includes is a step backwards. I now need to jump to 3 links to see which statements are supported. Also the fact that each of those pages have multiple sections it's very easy to misunderstand and think other sections are supported too.

Reply:
This was discussed in https://github.com/trinodb/trino/pull/8515#discussion_r670081746

The conclusion being that it's better to link out to a SQL statements overview page rather than duplicate common statements across connectors just to have the same content inline. For the sake of expediency & consistency I'd suggest we just move forward with this approach for now.

Reply:
Thanks for linking to the decision. I disagree with the conclusion but it's just an opinion. We can pick this up later.

Reply:
Feel free to mark my other comments in other PRs which link to this as resolved.